my-code/movie/src/components/like.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+
+export const Faved = () => {
```

Comment:
Minify the SVG, there are a lot of unnecessary code

Reply:
Always have a hard time around SVG...